docs/events.md

```diff
@@ -36,6 +36,7 @@ The following [inputs] are available in the `pull_request` context.
 | `dry-run`      | :heavy_check_mark: | :x:                |
 | `ignore`       | :heavy_check_mark: | :x:                |
 | `repo-token`   | :heavy_check_mark: | :heavy_check_mark: |
+| `strict`       | :heavy_check_mark: | :x:  |
```

Comment:
Could you change this so that the last `|` aligns with the `|` on the line before and after it? The same comment goes for the other changes as well.

For example:

```
| `repo-token`   | :heavy_check_mark: | :heavy_check_mark: |
| `strict`       | :heavy_check_mark: | :x:                |
| `svgo-config`  | :heavy_check_mark: | :x:                |
```